cranelift/codegen/meta/src/isa/x86/legalize.rs

```diff
@@ -790,6 +792,8 @@ fn define_simd(
     narrow.custom_legalize(ushr, "convert_ushr");
     narrow.custom_legalize(ishl, "convert_ishl");
     narrow.custom_legalize(fcvt_to_sint_sat, "expand_fcvt_to_sint_sat_vector");
+    narrow.custom_legalize(fmin, "expand_minmax_vector");
+    narrow.custom_legalize(fmax, "expand_minmax_vector");
```

Comment:
Can you use `expand_minmax`? If not, the reason is not clear from the doc comment on `expand_minmax_vector`.

Reply:
Oh, good point: the documentation doesn't make sense (copied from the `narrow_avx` instructions). I'll change it.

However, it still might make sense to keep this as a separate function from a code clarity perspective: these are completely different codegen approaches and `expand_minmax` is already complex enough as-is.